src/states/elder/mod.rs

```diff
@@ -663,11 +663,6 @@ impl Elder {
                 dst @ ManagedNode(_),
             ) => self.handle_connection_request(&encrypted_conn_info, pub_id, src, dst, outbox),
             (NeighbourInfo(_digest), ManagedNode(_), PrefixSection(_)) => Ok(()),
```

Comment:
Good start, but NeighbourInfo is dead code now, so I think we want to remove that too

Reply:
That was a mis-guided review comment. We do not want to remove NeighbourInfo, it is not dead code. The way we use it is that the source section if new can be added into their_keys.